spec/system/school/courses/inactive_students_index_spec.rb

```diff
@@ -63,5 +63,10 @@
     click_link 'Search'
     expect(page).to have_text(exited_team.founders.first.name)
     expect(page).to have_text(exited_team.name)
+
+    fill_in 'search', with: exited_team.founders.first.email
+    click_link 'Search'
+    expect(page).to have_text(exited_team.founders.first.name)
+    expect(page).to have_text(exited_team.name)
```

Comment:
It's good also to add a check that the list does not have a student who is not expected to be in the search result.